srcs/juloo.keyboard2/Config.java

```diff
@@ -324,6 +324,11 @@ private int getThemeId(Resources res, String theme_name)
       case "dark": return R.style.Dark;
       case "white": return R.style.White;
       case "epaper": return R.style.ePaper;
+        case "Lightv2": return R.style.Lightv2;
+        case "Darkv2": return R.style.Darkv2;
+        case "Amoled": return R.style.Amoled;
+        case "Desert": return R.style.Desert;
+        case "Jungle": return R.style.Jungle;
```

Comment:
The themes cannot be selected in the settings yet. You need to add them to `pref_theme_entries` and `pref_theme_values` in `res/values/arrays.xml`.

Reply:
In one of the changes I specified it, but I'll do it later xd, making the changes on the phone is complicated, that's why I love it XD